packages/datadog-instrumentations/src/jest.js

```diff
@@ -62,6 +68,7 @@ const specStatusToTestStatus = {
 
 const asyncResources = new WeakMap()
 const originalTestFns = new WeakMap()
+const retriedTestsToNumAttempts = new Map()
```

Comment:
`retriedTestsToNumAttempts` is used to track how many times a test has been retried